duckduckgo/iOS
# universal app link fix

Reviewer: Mia 
Asana: https://app.asana.com/0/414235014887631/430999069520960
CC:

**Description**:

This fixes a problem which occurs when a page redirects to a URL that has an associated universal link, launching the associated app and causing the page to finish with an error (load interrupted) which causes the error message UI to be shown.

(This does not solve the problem in which clicking on a search result from DDG results in a blank page as this is being fixed server side)

**Steps to test this PR**:

1. With the Twitter app installed, navigate to private.brindy.org.uk/twitter2.html
2. Click the link
3. The Twitter app will launch
4. Go back to the app and the previous page should be visible without any error message UI (note that the page may have scrolled up due to a separate issue, https://app.asana.com/0/414235014887631/416322909657029 )


###### Reviewer Checklist:


###### PR DRI Checklist:
